### PR TITLE
Forgot email pages

### PR DESCRIPTION
**Pull request by runimari**
Made pages for forgot email flow and added some classes to style. It's not connected to db yet though.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -118,9 +114,7 @@ func main() {
 	http.HandleFunc("/postresetpassword", forgotPasswordSent)
 	http.HandleFunc("/callback", callback)
 	http.HandleFunc("/employee-dashboard", employeeDashboard)
-	http.HandleFunc("/forgot-email", forgotEmail)
-	http.HandleFunc("/post-recovered-email", postRecoveredEmail)
-	http.HandleFunc("/verify-email-to-recover", verifyEmailtoRecover)
```
Comment (SharpBit) [inline main.go]: I think you accidentally got rid of some of your work with the merge

File: main.go [hunk 2]
```diff
@@ -409,121 +403,6 @@ func SetLoggedIn(w http.ResponseWriter, r *http.Request, attemptCookie *LogInAtt
...
-	http.ServeFile(w, r, "./templates/forgotemail.html")
-}
-
-func verifyEmailtoRecover(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/verifyemailtorecover.html")
-}
-
-func postRecoveredEmail(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/postrecoveredemail.html")
-}
```
Comment (SharpBit) [inline main.go:221]: This part too ^ Also, during the development of your code, we switched to using `RenderTemplate` similar to these:
https://github.com/MonopolyTechnic/simple-banking-system/blob/main/main.go#L131
https://github.com/MonopolyTechnic/simple-banking-system/blob/main/main.go#L175

**Review by SharpBit — COMMENTED** [review]
I think some of your work got deleted in the merge, also adduser.html seems to have the reset password html in it

**Review by SharpBit — APPROVED** [review]
lgtm

**Review by vasquezd21 — APPROVED** [review]
lgtm

**Pull request merged**